src/pudl/metadata/resources/eia.py

```diff
@@ -495,7 +495,10 @@
                 "previously_canceled",
             ],
             "primary_key": ["plant_id_eia", "generator_id"],
-            "foreign_key_rules": {"fields": [["plant_id_eia", "generator_id"]]},
+            "foreign_key_rules": {
```

Comment:
Why are we excluding epacamd_eia_subplant_ids here?

Reply:
ah good question! bc this table has some plant ids that don't actually come from eia so they are not in this ID table